### PR TITLE
Update .env

### PR DESCRIPTION
**Pull request by MaximMarkarov**
Test PULL

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -16,8 +16,8 @@
 ###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=ec360d80197652c1c092d3a7c6511e18
-#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
-#TRUSTED_HOSTS='^localhost|example\.com$'
```
Comment (Makcym) [inline .env]: what the hell?

File: .env [hunk 2]
```diff
@@ -16,8 +16,8 @@
 ###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=ec360d80197652c1c092d3a7c6511e18
-#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
```
Comment (Makcym) [inline .env]: Why?

Comment (Makcym) [reply .env]: Test**This**

**Comment by MaximMarkarov** [comment]
go go go

File: .env (hunk 1 above)
Comment (MaximMarkarov) [inline .env]: test 100

**Pull request merged**